### DISCUSSION TITLE
Global Settings Loading

### DISCUSSION THREAD

**Discussion by cemarta7** [lucky-media/landtamic#9] [General]
<img width="1146" alt="image" src="https://user-images.githubusercontent.com/5060049/170723062-d639f6cc-de65-40e0-b7d2-5b8514f2abf3.png">

Statamic 3.3.12
I try recompiling npm and the latest version.
When trying to change something then I get this error.

```
vue.common.prod.js:6 SyntaxError: Numeric separators are not allowed at the end of numeric literals
    at _default.jsonDecodeValue (Omitter.js:60:31)
    at Omitter.js:22:18
    at Array.forEach (<anonymous>)
    at _default.value (Omitter.js:21:25)
    at _default.value (Omitter.js:13:14)
    at a.visibleValues (HasHiddenFields.js:21:84)
    at fn.get (vue.common.prod.js:6:26596)
    at fn.evaluate (vue.common.prod.js:6:27746)
    at a.visibleValues (vue.common.prod.js:6:29650)
    at a.save (PublishForm.vue:201:1)
```

Any thoughts or ideas?

**Comment by lokmanm** [comment]
> <img alt="image" width="1146" src="https://user-images.githubusercontent.com/5060049/170723062-d639f6cc-de65-40e0-b7d2-5b8514f2abf3.png">
> 
> Statamic 3.3.12 I try recompiling npm and the latest version. When trying to change something then I get this error.
> 
> ```
> vue.common.prod.js:6 SyntaxError: Numeric separators are not allowed at the end of numeric literals
>     at _default.jsonDecodeValue (Omitter.js:60:31)
>     at Omitter.js:22:18
>     at Array.forEach (<anonymous>)
>     at _default.value (Omitter.js:21:25)
>     at _default.value (Omitter.js:13:14)
>     at a.visibleValues (HasHiddenFields.js:21:84)
>     at fn.get (vue.common.prod.js:6:26596)
>     at fn.evaluate (vue.common.prod.js:6:27746)
>     at a.visibleValues (vue.common.prod.js:6:29650)
>     at a.save (PublishForm.vue:201:1)
> ```
> 
> Any thoughts or ideas?

Hey @cemarta7,

I have seen this issue pop-up in the Statamic discord and I think its related to `compiled assets` not being copied over.

Our starter kit does not modify at all the `js/css` assets of the Statamic cp.

Can you try `composer dump-autoload` or removing the vendor folder and installing again.

**Reply by cemarta7** [reply]
I did try it. Same result.

I am able to not get the issue if I change the handler fields for the favicons from favicon_number to number_favicon.
![image](https://user-images.githubusercontent.com/5060049/170734528-4fc1122c-40a2-4cb5-a2d7-d7110a137281.png)
I could go and change the templates so they match these new values. 

I also had to change the 404_desc to textarea instead of bard. 
![image](https://user-images.githubusercontent.com/5060049/170734438-f9f1e1da-8bc2-47df-8930-c7a5b3755b33.png)

I also find out that if I turn off the toggle button I get this error:
```
vue.common.prod.js:6 SyntaxError: Delete of an unqualified identifier in strict mode.
    at _default.forgetValue (Omitter.js:87:14)
    at Omitter.js:30:18
    at Array.forEach (<anonymous>)
    at _default.value (Omitter.js:29:20)
    at _default.value (Omitter.js:14:14)
    at a.visibleValues (HasHiddenFields.js:21:84)
    at fn.get (vue.common.prod.js:6:26596)
    at fn.evaluate (vue.common.prod.js:6:27746)
    at a.visibleValues (vue.common.prod.js:6:29650)
    at a.save (PublishForm.vue:201:1)
```
Not sure if this will mess me up in the future but I can just bypass it like that for now.
Let me know if I am wrong.

**Reply by lokmanm** [reply]
@cemarta7

Hmm thats weird those field handles shouldn't be an issue. I will investigate this because I think its caused by a recent update of Statamic with the toggles, but I am not sure I will have to check.

**Reply by lokmanm** [reply]
@cemarta7 I found the issue.

The issue is with the `404_desc` not having a default container thats the reason it was throwing all those errors.

As soon as I added a default container it was fixed. 

I will push an update as soon as I can and you can copy directly the `yaml` blueprint directly from git.

Thanks
